bugzilla-dashboard/__init__.py

```diff
@@ -0,0 +1,51 @@
+from flask import Flask, jsonify, json
+import os
+from flask_cors import CORS
+from . import components
+
+app = Flask(__name__, static_folder='../static')
+SITE_ROOT = os.path.realpath(os.path.dirname(__file__))
+
+@app.errorhandler(404)
+def not_found(error):
+    return {
+        "status": 404,
+        "message": "Not found" 
+    }
+
+# Get component data
+@app.route('/data')
+def data():
+	if not os.path.exists('static'):
+		createDir()
+
+	componentsUrl = os.path.join(SITE_ROOT, "../static", "components.json")
+	if os.path.getsize(componentsUrl) > 0:
+		bzComponents = json.load(open(componentsUrl))
+		return bzComponents
+	else:
+		return 'No data found'
+
+# Update components data
+@app.route('/update')
```

Comment:
Ideally that update method should only be triggered by a CLI command.
I would prefer if you'd remove that endpoint, and made sure to be able to run `components.update` through CLI.

```shell
python3 bugzilla-dashboard/components.py 
```